aircraft/cost.py

```diff
@@ -0,0 +1,112 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
```

Comment:
In before @whoburg says "maybe that means we should check that those weird units have meaning"